src/icons/general/IconPuzzlePieceCircle.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import AccessibleSVG from '../accessible-svg';
+import { bdlBoxBlue } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    dimension?: number,
+    title?: string | React.Element<any>,
+};
+
+const IconPuzzlePieceCircle = ({ className = '', color = bdlBoxBlue, dimension = 28, title }: Props) => (
+    <AccessibleSVG
+        className={classNames('bdl-IconPuzzlePieceCircle', className)}
+        height={dimension}
+        viewBox="0 0 28 28"
+        title={title}
+        width={dimension}
+    >
+        <g fill="none" fillRule="evenodd" stroke={color}>
```

Comment:
you might not need the surrounding `<g>` if you move stroke={color} to the rect and path